modules/exploits/example_webapp.rb

```diff
@@ -168,6 +166,23 @@ def exploit
       'version' => '1.0',
       'uri' => '/' + ('../' * 560) + "\xcc\xcc\x90\x90" + '.smi'
     }, datastore['HttpClientTimeout'])
+
+    # example of sending a MIME message
+    data = Rex::MIME::Message.new
+    # https://github.com/rapid7/rex-mime/blob/master/lib/rex/mime/message.rb
+    file_contents = payload.encoded
+    data.add_part(file_contents, 'application/octet-stream', 'binary', "form-data; name=\"file\"; filename=\"uploaded.bin\"")
+    data.add_part('example', nil, nil, "form-data; name=\"_wpnonce\"")
```

Comment:
We technically have a cleaner API available nowadays that could be suggested, `vars_form_data`

https://github.com/rapid7/metasploit-framework/pull/16442